ios/RNPlayerView.swift

```diff
@@ -22,6 +22,7 @@ public class RNPlayerView: UIView {
     @objc var onStallStarted: RCTBubblingEventBlock?
     @objc var onStallEnded: RCTBubblingEventBlock?
     @objc var onTimeChanged: RCTBubblingEventBlock?
+    @objc var onPlaybackSpeedChanged: RCTBubblingEventBlock?
```

Comment:
nit: Since we don't follow any strict order here, it would make sense to move it as the last event-related property, right before `@objc var config: [String: Any]?`

Reply:
address in 1fa91bb913b00693336059784af698ad64254879